doc/audit/builtin_maps.rst

```diff
@@ -382,6 +382,15 @@ Governance history of the service, captures all COSE Sign 1 governance requests
 
 **Value** COSE Sign1
 
+``cose_recent_proposals``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Recent COSE proposal timeline, kept for the purpose of avoiding potential replay attacks.
+
+**Key** ccf.gov.msg.created_at field from COSE protect header, as a string, followed by SHA-256 digest of the COSE Sign1, represented as a hex-encoded string and separated by a ':'.
```

Comment:
The digest of the whole COSE message is not suitable for replay protection as the message is not canonical and can easily be modified without invalidating the signature. Simple example is adding an unprotected header parameter. Another example is using a non-optimal length encoding for one of the message elements (like the outer array length). 
I suggest using the signature or the hash of the signature instead.

Reply:
Thank you, that's a very good point.

Reply:
Switched to a digest of the signature.